nixos/modules/services/databases/ferretdb.nix

```diff
@@ -0,0 +1,79 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ferretdb;
+in
+{
+
+  meta.maintainers = with lib.maintainers; [ julienmalka camillemndn ];
```

Comment:
Does @camillemndn agree to be listed as maintainer?

Reply:
Yes, we worked together on the first draft of this PR ☺️